src/api/app/models/kiwi/image/xml.rb

```diff
@@ -0,0 +1,91 @@
+# TODO: Please overwrite this comment with something explaining the model target
+module Kiwi
+  class Image
+    class Xml
+      #### Includes and extends
+
+      #### Constants
+
+      #### Self config
+
+      #### Attributes
+
+      #### Associations macros (Belongs to, Has one, Has many)
+
+      #### Callbacks macros: before_save, after_save, etc.
+
+      #### Scopes (first the default_scope macro if is used)
+
+      #### Validations macros
+
+      #### Class methods using self. (public and then private)
+
+      #### To define class methods as private use private_class_method
+      #### private
+
+      #### Instance methods (public and then protected/private)
```

Comment:
Can you get rid of these comments because this is not an ActiveRecord model so they don't make sense here..